pkg/daemons/control/server.go

```diff
@@ -183,6 +183,9 @@ func apiServer(ctx context.Context, cfg *config.Control, runtime *config.Control
 	argsMap["requestheader-username-headers"] = "X-Remote-User"
 	argsMap["client-ca-file"] = runtime.ClientCA
 	argsMap["enable-admission-plugins"] = "NodeRestriction"
+	if cfg.DisableWatchCache {
+		argsMap["watch-cache"] = "false"
+	}
```

Comment:
why is this changed back and forth?
https://github.com/rancher/k3s/pull/32/commits/c0f49f3afc199ac098ab08ea80534038edec9868
https://github.com/rancher/k3s/pull/615

Reply:
Just testing things out.  It will be enabled when we finally ship.